backend/src/models/v2/Model.ts

```diff
@@ -38,6 +38,7 @@ export interface ModelInterface {
   id: string
 
   name: string
+  team: string
```

Comment:
Would a more accurate name be `teamId`?

Reply:
This isn't always a team ID. It's a select field with the ability to enter any freeform text. If the user selects something from the list it'll be an ID, otherwise it won't.

Reply:
What's the benefit of allowing users to enter any text they like? Will this not cause problems when managing the relationships between models and teams when the teams attached to models might or might not be teams that exist within BAILO?

Reply:
After discussion we've decided to got with teamId. Changed!